routes/api.js

```diff
@@ -159,6 +160,66 @@ router.patch('/user/badge/update', authenticationHelpers.isAuthOrRedirect, (req,
     }
 });
 
+router.post('/user/activity/levelNumber', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    if (!req.user.email) {
+        return res.status(400).json({error: errorCode.PCE021.message, code: errorCode.PCE021.code});
+    }
+    localUser.find({where: {
+        email: req.user.email
+    }}, {raw: true})
+        .then(data => {
+            if (!data) {
+                return res.status(402).json({info: errorCode.PCE022.message, code: errorCode.PCE022.code});
+            }
+            levelProgress.find({where: {
+                user_id: data.id
+            }}, {raw: true})
+                .then(activityData => {
+                    if (!activityData) {
+                        return res.status(402).json({info: errorCode.PCE036.message, code: errorCode.PCE036.code});
```

Comment:
402 is again used? I don't think its right

Reply:
yes it should be 400 as in the whole app.

Reply:
@shreyans29 I think it should be 200 only, the same is being discussed in the next PR.

Reply:
@shreyans29, @Raxerz may be correct here. Because **200** actually means that **if** block worked properly. So, for a Developer, this would mean that exceptions are handled correctly and hence success.